src/Arr.php

```diff
@@ -613,6 +613,17 @@ public static function isArray(mixed $input): bool
         return is_array($input);
     }
 
+    /**
+     * Returns true if the input array is empty
+     *
+     * @param  array  $input the array value, to test if it is an empty array
+     * @return bool true if $input is an empty array, false otherwise
+     */
+    public static function isEmpty(array $input): bool
```

Comment:
Wouldn't you be able to do the same just using the `make` method and `count` method while casting it to a boolean?

```php
use HiFolks\DataType\Arr;
$result = (bool) Arr::make([])->count();
```

Reply:
🤔 

Maybe changing the method to be:
```php
public function isEmpty(): bool
{
	return $this->length() == 0;
}
```

This way we could change it after commands like `reduce` and `filter` .

```php
use HiFolks\DataType\Arr;
$result = Arr::make([1, 2, 3])
	->filter(fn ($element) => $element % 2)
	->isEmpty();
```

Reply:
Well, yeah, I could, but I would prefer to have one specific method for checking if array is empty and not calling other methods. I mean, in my opinion, it would be more clear and convenient to understand that this func is checking if array is empty or not

Reply:
@RoadSigns I agree with your last proposal. I could adjust the code accordingly

Reply:
amazing conversation @Tautve  and @RoadSigns ! I agree that having a specific and chainable method just for checking if the Arr is empty is good for the Developer Experience. Thank you, everyone!!!

Reply:
The code for me is fine. Can i merge it  @Tautve ?

Reply:
@roberto-butti I don't think the changes have been pushed yet 👍

Reply:
@roberto-butti I'm still working on it, it'll take a few minutes

Reply:
@roberto-butti PR is updated

Reply:
great job!